### ISSUE TITLE
Uncaught Error: Tried to unload active package 'github'

### ISSUE THREAD

**Issue by spideep** [atom-community/sync-settings#449]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.33.0-beta3 x64
**Electron**: 2.0.11
**OS**: Ubuntu 18.04.1
**Thrown From**: [sync-settings](https://github.com/atom-community/sync-settings) package 0.8.6

### Stack Trace

Uncaught Error: Tried to unload active package 'github'

```
At /usr/share/atom-beta/resources/app/static/<embedded>:11

Error: Tried to unload active package 'github'
    at PackageManager.unloadPackage (/usr/share/atom-beta/resources/app/static/<embedded>:11:381402)
    at PackageManager.module.exports.PackageManager.install (/packages/sync-settings/lib/package-manager.coffee:158:19)
    at Object.installPackage (/packages/sync-settings/lib/sync-settings.coffee:439:20)
    at /packages/sync-settings/lib/sync-settings.coffee:410:12
    at /packages/sync-settings/lib/sync-settings.coffee:409:9
    at Object.installMissingPackages (/packages/sync-settings/lib/sync-settings.coffee:433:7)
    at /packages/sync-settings/lib/sync-settings.coffee:246:16
    at /packages/sync-settings/node_modules/github/lib/index.js:819:17
    at callCallback (/packages/sync-settings/node_modules/github/lib/index.js:667:17)
    at /packages/sync-settings/node_modules/github/lib/index.js:726:25)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1056:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

### Commands

```
     -0:30.7.0 sync-settings:restore (atom-workspace.workspace.scrollbars-visible-always.atom-ui-tweaks--tab--border-radius--large.atom-ui-tweaks--tab--separator-style--slight.atom-ui-tweaks--ui--atom-logo-splash-screen.theme-base16-tomorrow-light-theme.theme-one-dark-ui)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
Atom-SFTP-sync 2.0.1 
atom-terminal 0.8.0 
atom-ui-tweaks 0.3.0 
auto-reveal-active-file 0.3.1 
better-git-blame 0.4.7 
busy-signal 1.4.3 
color-picker 2.3.0 
docker 0.8.0 
file-icons 2.1.26 
git-control 0.9.0 
git-gui 0.1.27 
git-plus 8.1.0 
git-time-machine 2.1.0 
github-utils 1.6.0 
Handlebars 1.8.0 
intellij-idea-keymap 0.2.3 
intentions 1.1.5 
jQuery 1.2.6 
jslint 1.5.1 
linter 2.2.0 
linter-scss-lint 3.1.1 
linter-ui-default 1.7.1 
min-ui 0.0.4 
minimap 4.29.9 
minimap-git-diff 4.3.1 
numix-ui 0.2.3 
perfectionist 2.0.0 
php-twig 4.0.0 
pigments 0.40.2 
pinned-tabs 2.1.0 
react 0.18.0 
remote-ftp 2.2.2 
scroll-percentage 1.3.0 
SFTP-deployment 1.0.2 
split-diff 1.5.3 
svg-preview 0.14.0 
sync-settings 0.8.6 
tree-view-git-status 1.5.2 
vaporwave-ui 0.0.2 
youtube-pane 2.0.0 
```

**Comment by WyRe** [comment]
I just had this issue, I've solved it disabling manually `github` built-in package (in settings -> packages tab) and then restoring the backup typing `sync settings restore` in the command palette (`ctrl + alt + p`) 

The backup was successfully applied. And also sync-settings enables again `github` built-in package automatically.

**Comment by WyRe** [comment]
This issue looks like produced by `sync-settings`, apparently is adding a core package to `packages.json` this is wrong cause core/built-in packages are upgraded in Atom's upgrades.

Finally I've solved the issue manually removing that entry from my gist. More info [in here](https://discuss.atom.io/t/why-is-not-atom-managing-properly-github-built-in-package-update/63041/4)

**Comment by UziTech** [comment]
should be fixed by v2.0.0
